shared-ui/components/footer/Footer.tsx

```diff
@@ -31,7 +31,7 @@ const Footer: React.FC<FooterProps> = ({ tabs, isDay }) => {
           <StyledFooterText>
             HackBeanpot, Inc. is a registered 501(c)(3) organization.
           </StyledFooterText>
-          <StyledFooterTextLink href="https://hackbeanpot.com/code-of-conduct/">
+          <StyledFooterTextLink href='/code-of-conduct'>
```

Comment:
we probably shouldn't link to the prod site from dev/vercel - this is the change that makes this PR add another copy of the code of conduct page to the live site. might be a better way to do this with gatsby?

Reply:
I think we have it like this bc David / Sreya wanted the code-of-conduct page to just be on the main site instead of having the page also on the live-site